app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+
+  def index
+    rents = Rent.all
```

Comment:
Not a correction right now but remember not to use index methods that just render Model.all unless you know that's a really small table with a fixed number of rows or something loke that.